build.proj

```diff
@@ -89,18 +87,16 @@
   <!-- Hook that can be used to insert custom build tasks to the build process such as setup and/or cleanup tasks -->
   <Import Project="build.override.targets" Condition="Exists('build.override.targets')" />
 
-  <Target Name="UpdateVSConfigurations">
-    <Message Importance="High" Text="Updating configurations for projects ..." />
+  <!-- set up inputs for UpdateVSConfigurations in a target to avoid globbing all the time -->
+  <Target Name="_setProjectsToUpdate" BeforeTargets="UpdateVSConfigurations">
```

Comment:
To do an update of all the repo I guess?

Reply:
Yes, it supports UpdateVSConfigurations from build.proj.  We have a target to defer item evaluation until that target is UpdateVSConfigurations invoked, so that we don't end up wasting folks time with the globbing on a regular build.

Reply:
Cool I just learned something. That's a neat technique!